ptn/boozebot/commands/Cleaner.py

```diff
@@ -97,6 +97,63 @@ async def booze_channels_close(self, ctx: SlashContext):
 
         await ctx.send(f"<@&{server_sommelier_role_id()}> That\'s the end of that, me hearties.", embed=embed)
 
+    @cog_ext.cog_slash(
+        name="Clear_Booze_Roles",
+        guild_ids=[bot_guild_id()],
+        description="Removes all WC/Hitchhiker users. Requires Admin/Mod/Sommelier - Use with caution.",
+        permissions={
+            bot_guild_id(): [
+                create_permission(server_admin_role_id(), SlashCommandPermissionType.ROLE, True),
+                create_permission(server_sommelier_role_id(), SlashCommandPermissionType.ROLE, True),
+                create_permission(server_mod_role_id(), SlashCommandPermissionType.ROLE, True),
+                create_permission(bot_guild_id(), SlashCommandPermissionType.ROLE, False),
+            ]
+        },
+    )
+    async def clear_booze_roles(self, ctx: SlashContext):
+        """
+        Command to reset the Wine Carrier and Hitchhiker roles to have no members. Generates a message in the channel that it ran in.
+
+        :param SlashContext ctx: The discord slash context.
+        :returns: A discord embed
+        """
+        print(f'User {ctx.author} requested clearing all Booze related roles in channel: {ctx.channel}.')
+
+        guild = bot.get_guild(bot_guild_id())
+        role_id = server_wine_carrier_role_id()
+        role = discord.utils.get(ctx.guild.roles, id=role_id)
+        count = 0
+        try:
+            for member in guild.members:
+                if role in member.roles:
+                    try:
+                        await member.remove_roles(role)
+                        count += 1
+                    except Exception as e:
+                        print(e)
+                        await ctx.send(f"Unable to remove { role } from { member }")
+            await ctx.send(f'Successfully removed { count } users from the Wine Carrier role.')
+        except Exception as e:
+            print(e)
+            await ctx.send('Clear roles command failed.  Contact admin.')
+
+        count = 0
```

Comment:
These could be combined into a single function - make a list of role_ids and iterate through the function for each role_id in the list.